fhir-server/liberty-config/configDropins/disabled/jwtRP.xml

```diff
@@ -0,0 +1,32 @@
+<server description="fhir-server">
+
+    <!-- Enable features -->
+    <featureManager>
+        <!-- mpJwt-1.1 is already enabled in the default server.xml, but it doesn't hurt to repeat it here -->
+        <feature>mpJwt-1.1</feature>
+    </featureManager>
+
+    <!-- Override the application-bnd binding of the main webapp -->
+    <webApplication contextRoot="fhir-server/api/v4" id="fhir-server-webapp" location="fhir-server.war" name="fhir-server-webapp">
+        <application-bnd>
+            <security-role id="users" name="FHIRUsers">
+                <group name="JwtUsers" access-id="group:https://localhost:9443/oauth2/endpoint/oauth2-provider/FHIRUsers"/>
+            </security-role>
+        </application-bnd>
+    </webApplication>
+
+    <!-- The MP JWT configuration that injects the caller's JWT into a
+         ResourceScoped bean for inspection. -->
+    <mpJwt id="jwtConsumer"
+           keyName="libertyop"
+           userNameAttribute="sub"
+           groupNameAttribute="groupIds"
+           issuer="https://localhost:9443/oauth2/endpoint/oauth2-provider,https://host.docker.internal:9443/oauth2/endpoint/oauth2-provider"
+           authFilterRef="filter"/>
+
+    <authFilter id="filter">
+        <requestUrl urlPattern="/fhir-server" />
+        <requestUrl matchType="notContain" urlPattern="/fhir-server/api/v4/metadata" />
```

Comment:
posted a message about implications for opening up... is this cached?

Reply:
we have https://github.com/IBM/FHIR/issues/734 for caching of metadata

it doesn't connect to the database, but it does read the config (both the tenant config and the default config), and programatically builds up the capabilitystatement, which can take some.